src/components/cards/CardItem.vue

```diff
@@ -135,6 +139,9 @@ export default {
 			const board = this.$store.getters.boards.find((item) => item.id === this.card.boardId)
 			return board ? !board.archived && board.permissions.PERMISSION_EDIT : false
 		},
+		inlineEditingBlocked() {
+			return compactMode || isArchived || showArchived || !canEdit || standalone
```

Comment:
I think `compactMode`, `isArchived`, `showArchived`, `canEdit`, `standalone` variables are undefined as you can see in the github action.

Reply:
Ah, good catch